### PR TITLE
Profile Total View Count

### PR DESCRIPTION
**Pull request by steam**
Adds the UI for Total View Count. User interaction and category modal will be a separate PR.

profiletotalcountview_on_custom_sized__375_0__60_0_ 2x

### CODE REVIEW

**Comment by steam** [comment]
Just realized that the category badge will need to be hidden when the user is not featured in any categories. I'll include that in a new PR that includes the featured category modal.

File: Sources/Model/User.swift [hunk 1]
```diff
@@ -10,6 +10,7 @@ import SwiftyJSON
 // version 3: added notifyOfWatchesViaPush, notifyOfWatchesViaEmail
 // version 4: added notifyOfCommentsOnPostWatchViaPush, notifyOfCommentsOnPostWatchViaEmail
 // version 5: added isCollaborateable, moved notifyOf* into Profile
+// version 6: added totalViewCount, categories
```
Comment (colinta) [inline Sources/Model/User.swift]: Mentions `categories`, but they're not added here.

Comment (steam) [inline Sources/Model/User.swift]: I'll kill it for now.

Comment (steam) [inline Sources/Model/User.swift]: Killed it.

**Pull request merged**